roles/ceph-validate/tasks/main.yml

```diff
@@ -1,33 +1,35 @@
 ---
-- name: validate ceph_origin
-  fail:
-    msg: "ceph_origin must be either 'repository', 'distro' or 'local'"
-  when:
-    - not containerized_deployment | bool
-    - ceph_origin not in ['repository', 'distro', 'local']
+- name: validate repository variables in non-containerized scenario
+  when: not containerized_deployment | bool
+  block:
+    - name: validate ceph_origin
+      fail:
+        msg: "ceph_origin must be either 'repository', 'distro' or 'local'"
+      when:
+        - ceph_origin not in ['repository', 'distro', 'local']
```

Comment:
I would prefer to have the condition on one line there

Reply:
Done!
Thank you